packages/prepo-ui/src/components/Simulator/Simulator.tsx

```diff
@@ -50,84 +52,64 @@ const TrackProgress = styled.div`
   position: absolute;
 `
 
+export function mapDistanceToPrice({
+  data,
+  direction,
+  distance,
+}: {
+  data: SimulatorDataProps | undefined
+  direction: 'long' | 'short'
+  distance: number | undefined
+}): number | undefined {
+  if (!data || distance === undefined) return undefined
+  const position = direction === 'long' ? distance : 1 - distance
```

Comment:
probably don't want to do this here because this distance is already "reverted" (e.g. regardless of direction, best entry is always 0)

So if I'm shorting, and simulating with entry at best rate (distance: 0), I want to get payout at payout[0], if you do 1 - distance here I will get payout[1]


```suggestion
  const position = direction === 'long' ? distance : 1 - distance
```